wandb/apis/public/runs.py

```diff
@@ -161,6 +161,55 @@ def convert_objects(self):
 
         return objs
 
+    @normalize_exceptions
+    def histories(
+        self, samples=500, keys=None, x_axis="_step", pandas=True, stream="default"
+    ):
+        """Return sampled history metrics for all runs that fit the filters conditions.
+
+        Arguments:
+            samples : (int, optional) The number of samples to return per run
+            pandas : (bool, optional) Return a pandas dataframe
```

Comment:
I would use `format: Literal["default", "pandas", "polars"] = "default"`
(and type-annotate stream similarly)

Reply:
thank you @dmitryduev! `format` and `stream` are now type-annotated

Reply:
@dmitryduev test has been added as well